Makefile

```diff
@@ -222,12 +222,6 @@ migrate:
 dbchange:
 	$(MANAGEMENT_COMMAND) makemigrations
 
-supervisor:
-	@if [ "$(VENV_BASE)" ]; then \
-		. $(VENV_BASE)/awx/bin/activate; \
-	fi; \
-	supervisord --pidfile=/tmp/supervisor_pid -n
```

Comment:
I've grown kind of tired of this pattern of make-calls-docker, then docker-calls-supervisord-entrpoint, then supervisord-calls-make.

While we could get more explicit about documenting "these targets are ran inside docker" vs "these targets are ran outside docker", in this case it makes more sense to just delete the unnecessary step.